wordpress-mobile/WordPress-iOS
# Enable page templates

This PR is only a helper to get installable builds, there's no need to merge these changes outside the regular Gutenberg release process.

gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1987
gutenberg PR: https://github.com/WordPress/gutenberg/pull/20718
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/13595
WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/11409

To test:

Create a new page and verify that the template picker is showing

PR submission checklist:
